server/commands/userCreator.ts

```diff
@@ -97,22 +105,12 @@ export default async function userCreator({
         [Op.is]: null,
       },
     },
-    include: [
-      {
-        model: UserAuthentication,
-        as: "authentications",
-        required: false,
-      },
-    ],
   });
 
   // We have an existing invite for his user, so we need to update it with our
   // new details and link up the authentication method
   if (invite && !invite.authentications.length) {
-    const transaction = await User.sequelize!.transaction();
```

Comment:
Also moving from this old transaction format which requires the `!` where they're near code that's being updated